src/Core/CommandBus/ExecutedCommandRegistry.php

```diff
@@ -84,16 +90,42 @@ public function register($command, $handler)
     /**
      * @return array
      */
-    public function getExecutedCommands()
+    public function getExecutedCommands(): array
     {
         return $this->registry['commands'];
     }
 
     /**
      * @return array
      */
-    public function getExecutedQueries()
+    public function getExecutedQueries(): array
     {
         return $this->registry['queries'];
     }
+
+    /**
+     * Returns the file and line that invoked the handle method
+     *
+     * @return array
+     */
+    private function getTrace(): array
+    {
+        $trace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 10);
```

Comment:
```suggestion
        $trace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, self::BACKTRACE_LIMIT);
```

Sorry, I hate aloned number :sweat_smile:

Reply:
Done